sql/analyzer/check_constraints.go

```diff
@@ -142,8 +142,9 @@ func loadChecks(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope) (sql.No
 
 			return &nn, nil
 		case *plan.Update:
-			rtable := getResolvedTable(node)
+			nn := *node
```

Comment:
Fine to move this up but replace all the `node` references below with `nn`.

Same with other blocks

Reply:
✅